src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs

```diff
@@ -325,10 +325,21 @@ private bool IsValidationRequired(HttpContext httpContext)
             return true;
         }
 
+        private static IAntiforgeryContextAccessor GetAntiforgeryFeature(HttpContext httpContext)
```

Comment:
Get rid of `IAntiforgeryContextAccessor` - rename `AntiforgeryContext` -> `IAntiforgeryFeature`